docs/network-requirements.md

```diff
@@ -15,6 +15,7 @@ firewalls allow the necessary traffic based on your configuration.
 | Antrea with feature BGPPolicy enabled          | Selected by user-provided BGPPolicies | TCP 179<sup>[1]</sup>                      | Yes          |                              |
 | All                                            | Kube-apiserver host                   | TCP 443 or 6443<sup>[2]</sup>              | Yes          |                              |
 | All                                            | All                                   | TCP 10349, 10350, 10351, UDP 10351         | Yes          |                              |
+| Antrea with proxyAll enabled                   | All                                   | TCP 10256<sup>[4]</sup>                    | Yes          |                              |
```

Comment:
Shouldn't we mention that this is "optional"? AFAIK, this is mostly needed for integration with external LBs?